docs/optimization_guide/nncf/ptq/code/ptq_torch.py

```diff
@@ -15,7 +15,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # torch.nn.Module object
+import torch.nn as nn
+model = nn.Module() # torch.nn.Module object
```

Comment:
PyTorch model can not be loaded from the file, thus I would suggest to stay the previous version.